### ISSUE TITLE
Unable to use guppy using webpack

### ISSUE THREAD

**Issue by nishantwrp** [daniel3735928559/guppy#181]
I'm unable to load Guppy with `build/guppy.js` and `build/guppy.min.js` using webpack.

**Comment by jalonsomartinez** [comment]
Same here.
¿Did you find a solution?
Trying to integrate Guppy in Vuejs

**Comment by nishantwrp** [comment]
@jalonsomartinez Yup, make your own fork of this repository and do these changes. https://github.com/oppia/guppy/commit/7509f31c3f6e1205763d605ab41098cf6ac9d738
